emailpassword/docs/common-customizations/sessions/user-information-front-end.md

```diff
@@ -49,6 +49,17 @@ render() {
 }
 ```
 
+### Automatic context updates
+`SessionContext` will be updated for `EmailPasswordAuth` children whenever any of the following events happens:
```

Comment:
This is too much information provided in this recipe docs.. it's good for auth-react SDK docs.. but in recipe, we don't need this.